app/models/miq_report_result.rb

```diff
@@ -36,7 +36,7 @@ def status
   end
 
   def status_message
-    miq_task.nil? ? _("Report results are no longer available") : miq_task.message
+    miq_task.nil? ? _("Task associated with this report no longer available") : miq_task.message
```

Comment:
I believe this is logged or in the UI so, maybe `The task associated with this report is no longer available"`